config/config.go

```diff
@@ -0,0 +1,33 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/tkanos/gonfig"
+)
+
+type Configuration struct {
```

Comment:
By convention, all exported functions and structs definitions should contain a comment in the line above them to describe what is the purpose of that. 

Starting with the function/struct name, following by a short description.

```suggestion
// Configuration struct to store project configuration
type Configuration struct {
```

Please, add a comment to each function.